protocol/implementations/js/src/protocol-definitions.ts

```diff
@@ -37,7 +37,7 @@ export const aliceProtocolDefinition = {
           $actions: [
             {
               who : 'recipient',
-              of  : 'RFQ/Quote',
+              of  : 'RFQ',
```

Comment:
request: could you get rid of the comment on L35? 🙏 i forgot to get rid of it when i got rid of Order

specifically this line: 
so i think here we want to say `OrderStatus can be written to Alice's DWN by someone who wrote RFQ/Quote (i.e. PFI)`

```js
$actions: [
            {
              who : 'author',
              of  : 'RFQ/Quote',
              can: 'write'
             }]
```

or `OrderStatus can be written to Alice's DWN by some who received an RFQ

```js
$actions: [
            {
              who : 'recipient',
              of  : 'RFQ',
              can: 'write'
             }]
```

looks like you went with the second option which makes sense to me, i think the original was wrong :)

Reply:
oh yeah... you know what, we should do this instead... notice how I'm using the `author` of the `RFQ/Quote`, which is your first comment proposal

```typescript
// OrderStatus can be written to Alice's DWN by someone who wrote RFQ/Quote (i.e. PFI)
OrderStatus: {
  $actions: [
    {
      who : 'author',
      of  : 'RFQ/Quote',
      can : 'write'
    }
  ]
}
```

done